config/systemd/omnect-ui.tmpfilesd

```diff
@@ -0,0 +1 @@
+d /var/lib/omnect-ui 0750 omnect_device_socket omnect_device_socket -
```

Comment:
not sure: do we want to continue with this name or rename to omnect_device_service_client?

Reply:
I would like to rename but would also like to discuss potential results regarding compatibility first

@ronny-standtke : FYI